README.md

```diff
@@ -3,19 +3,29 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T848441)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-# How to implement CRUD operations via API controllers in an ASP.NET Core with Razor Pages project
 
-This example demonstrates how to bind dxDataGrid to a controller action and implement CRUD operations via an API controller in a Razor Pages project.
-Set the Controller, LoadAction, UpdateAction, InsertAction and DeleteAction options so that dxDataGrid's data source can perform CRUD operations. In the corresponding controller, implement data operations in the corresponding actions.
+# DataSource for DevExtreme - How to implement CRUD operations with API controllers in ASP.NET Core in a Razor Pages project
```

Comment:
What do you think about the following alternative?

`# DataSource for DevExtreme - How to implement CRUD operations with API controllers in an ASP.NET Core Razor Pages project`

Reply:
Noted